components/bokeh/__init__.py

```diff
@@ -0,0 +1,67 @@
+from bokeh.plotting import figure, show
+from bokeh.transform import factor_cmap
+from bokeh.models import ColumnDataSource, Select, NumeralTickFormatter
+from bokeh.models.callbacks import CustomJS
+from bokeh.layouts import column, row
+import numpy as np
+import seaborn as sns
+
+def generate_colors(input_df, feature):
+    pal = sns.color_palette()
+    color = factor_cmap(feature, palette=pal.as_hex(), factors=np.array(input_df[feature].unique()))
```

Comment:
I don't believe the outer `np.array(...)` here is necessary

Reply:
We need some ways to remove dtype=object that is generated by unique().
I changed it to be input_df[feature].unique().tolist()